app/controllers/v0/letters_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module V0
+  class LettersController < ApplicationController
+    def index
+      response = EVSS::Letters::Letter.find_by_user(@current_user)
+      render json: response.as_json
```

Comment:
The abstractions look a little funky.  Typically, when I see `<MODEL>.find_by(x)`, I think of the response as that `<MODEL>`, e.g. `car = Car.find_by_plate('abc-123')`, but in your case it's `response`.  I think I'd prefer either

```ruby
raw_letters_response = LetterService.get_letters_for_user(@current_user)
```
or
```ruby
letters = Letters.find_by_user(@current_user)
```
but what you have now is a combination of the two.